packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -92,6 +92,26 @@ class SlidePanel extends React.Component {
       customProps.className,
     ]);
 
+    const content = (panelPosition === 'start') ? (
+      <React.Fragment>
+        <div className={cx(['panel'])} tabIndex="-1" aria-hidden={!isOpen ? 'true' : 'false'} ref={this.setPanelNode}>
+          {panelContent}
+        </div>
+        <div className={cx('main')} tabIndex="-1" ref={this.mainNode}>
+          {mainContent}
+        </div>
```

Comment:
we can declare these as two separate constants and call those one in both if and else. do avoid code duplication.

Reply:
Made the appropriate changes of declaring the two separate constants for panel and main